scripts/build_message_web.py

```diff
@@ -0,0 +1,146 @@
+import os, time, argparse
+import json
+from elasticsearch import Elasticsearch
+from datetime import datetime
+import classes.class_generate_events_json as calendar_events
+
+import support as support
+
+def fetch_current_indexed_sdk_files(es : Elasticsearch, index_name='github_live_index'):
```

Comment:
Index name value as secret.

Reply:
Added it as ${{ secrets.ES_INDEX_LIVE }}.